glean-core/android-native/build.gradle

```diff
@@ -0,0 +1,117 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// This is based off:
+// https://github.com/mozilla/application-services/blob/c40e2ccb422cf4af9ffdf095149cec34de1d4bef/components/fxa-client/android/build.gradle
+
+import groovy.json.JsonOutput
+
+apply plugin: 'com.android.library'
+apply plugin: 'org.mozilla.rust-android-gradle.rust-android'
+
+android {
+    compileSdkVersion rootProject.ext.build.compileSdkVersion
+
+    defaultConfig {
+        minSdkVersion rootProject.ext.build['minSdkVersion']
+        targetSdkVersion rootProject.ext.build['targetSdkVersion']
+
+        // TODO: 1551691 Get the version from git tag...? Also, we need to select a
+        // version that won't conflict with legacy glean-ac versions.
+        buildConfigField("String", "LIBRARY_VERSION", "\"0.1\"")
```

Comment:
Should we use the same numbers that we use for the rest of the SDK?

Reply:
This is copied. I don't think we _need_ it anymore really.